File: 0378-kth-smallest-element-in-a-sorted-matrix/0378-kth-smallest-element-in-a-sorted-matrix.cpp
class Solution {
public:
    int kthSmallest(vector<vector<int>>& matrix, int k) {
         vector<int> list;
        int r = matrix.size();       
        int c = matrix[0].size();  
        for(int i =0; i<r;i++){
            for(int j=0; j< c; j++){
                 list.push_back(matrix[i][j]);
            }
        }
        sort(list.begin(), list.end());
        return list[k-1];
    }
};

File: 0378-kth-smallest-element-in-a-sorted-matrix/0378-kth-smallest-element-in-a-sorted-matrix.java
class Solution {
    public int kthSmallest(int[][] matrix, int k) {
        List<Integer> list = new ArrayList<>();
        int r = matrix.length;
        int c = matrix[0].length;
        for(int i=0; i<r; i++){
            for(int j=0; j<c; j++){
               list.add(matrix[i][j]);
            }
        }
        Collections.sort(list);
       
        return list.get(k-1);
    }
}